example-local.yaml

```diff
@@ -3,14 +3,41 @@ packages:
   econet: !include
     # Change this file to the correct one for your device
     file: econet_heatpump_water_heater.yaml
+    # next line is needed for all hvac single zone installs
     # file: econet_hvac.yaml
+    # next line is needed for all hvac multi zone installs (do not include above)
+    # file: econet_hvac_base.yaml
     # Use this if you have a communicating outdoor unit (ODU)
     # files: [econet_hvac.yaml, econet_hvac_odu.yaml]
     # file: econet_tankless_water_heater.yaml
     # file: econet_electric_tank_water_heater.yaml
     vars:
       external_components_source: components
 
+  # for zones - include the econet hvac package above and the appropriate zones below
+  # because we are re-using esphome-econet for each inclusion, the path is required also.
+  # zone1: !include
+  #    url: https://github.com/esphome-econet/esphome-econet
+  #   file: econet_hvac_zone_template.yaml
+  #   vars:
+  #     zoneid: "1"
+  #     zonename: "Zone1"
+  #     requestmod: 5
+  # zone2: !include
+  #    url: https://github.com/esphome-econet/esphome-econet
+  #   file: econet_hvac_zone_template.yaml
+  #   vars:
+  #     zoneid: "2"
+  #     zonename: "Zone2"
+  #     requestmod: 6
```

Comment:
Do you really need to increment request_mod?

Reply:
different request mods are required in new code due to different src addresses of zone thermostats